src/box/error.cc

```diff
@@ -80,6 +80,7 @@ box_error_set(const char *file, unsigned line, uint32_t code,
 	struct error *e = BuildClientError(file, line, ER_UNKNOWN);
 	ClientError *client_error = type_cast(ClientError, e);
 	client_error->code = code;
+	client_error->set_name();
```

Comment:
I don't like this new method. Why can't we use the proper `ClientError` constructor instead?